tools/automate_aws_os_version.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import os
+import json
+import re
+import subprocess
+OS_COMMANDS = ['aws ec2 describe-images --owners amazon --filters "Name=name,Values=*Windows_Server-2019-English-Full-Base-*" --query "sort_by(Images, &CreationDate)[].Name"',
```

Comment:
i would probably structure this differently. Have a list of dict, like:
[ {"owner": "amazon", "name": "*Windows_Server-2019-English-Full-Base-*" },
 {"owner": "099720109477", "name": "*ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-*"},
...
]

then just define the command once and apply string formatting to get the actual command.